eras/shelley/impl/src/Cardano/Ledger/Shelley/TxBody.hs

```diff
@@ -214,7 +214,7 @@ basicShelleyTxBodyRaw =
     { stbrInputs = Set.empty
     , stbrOutputs = StrictSeq.empty
     , stbrTxFee = Coin 0
-    , stbrTTL = SlotNo 0
+    , stbrTTL = SlotNo maxBound -- transaction is eternally valid by default
```

Comment:
The reason for this change is to make it consistent with Allegra onwards, which makes transactions with valid and permissive interval by default: https://github.com/IntersectMBO/cardano-ledger/blob/411054e40b4e08350049e4eaffdf0cc5f73e9d91/eras/allegra/impl/src/Cardano/Ledger/Allegra/TxBody.hs#L197

> Is there no potential for problems since we are loosening in a way the conditions?

Not really, since no-one constructs Shelley transactions anymore, except the core team for testing.

It was already in a separate commit 84424186558e45e9dff5419c47d9b99db0c3b590